### ISSUE TITLE
Bleed box around “computer sind scheisse”

### ISSUE THREAD

**Issue by jaseg** [mwarning/chaos-sticker-collection#36]
Hey there,

I saw you have a copy of one of my sticker designs, the pink square “computer sind scheisse” sticker. I noticed that in the web thingy the design doesn’t show up properly because the files are exported with a 3mm bleed box for printing (this allows for some tolerance when cutting so you don’t get white flashing at the edges). If you want to display them on screen, you have to cut off that bleed box.

just thought I’d tell you :)
- jaseg

**Comment by mwarning** [comment]
hi @jaseg, I do not see that this is issue (but I am not a physical sticker expert). Wouldn't anyone who wants to print it adjust the border for their needs?

**Comment by jaseg** [comment]
@mwarning yeah for print it’s fine as long as you know it’s there, but in the web thing it looks wrong because the composition of the design is off.

**Comment by mwarning** [comment]
There is a notes with the sticker that there is a 3mm bleeding box included:
That is all I can do: https://github.com/mwarning/chaos-sticker-collection/commit/94f9e1952cbd120b41f0d8408336ee999ae10132
